ui/js/component/tipLink/view.jsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import Link from "component/link";
+import FormFieldPrice from "component/formFieldPrice";
+
+class TipLink extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      feeAmount: "1.00",
+      currency: "LBC",
+    };
+  }
+
+  handleSupportButtonClicked() {
+    this.resetDefaults();
+    this.props.onTipShow();
+  }
+
+  handleSendButtonClicked() {
+    let claim_id = this.props.claim_id;
+    let amount = this.state.feeAmount;
+
+    this.props.setClaimID(claim_id);
+    this.props.setAmount(amount);
+    this.props.sendSupport();
```

Comment:
Amount's value in store is needed for [this](https://github.com/lbryio/lbry-app/pull/391/files#diff-10945e96d61f083b2a59634a088e8c7aR16)

As for `claim_id`; currently it being passed as a prop [here](https://github.com/lbryio/lbry-app/pull/391/files#diff-1fa5cb7e26763162ff46465cbb589c6eR190), which was possible due to having the `claimInfo` in FileActions, but since this component will be moved to `FilePage`, `claim_id`(for support) will be set during the loading of the page by directly fetching the values from the store(`FETCH_FILE_INFO_COMPLETED` action to be specific). Seems to me a neater and React way of doing this. Also it will ensure no redundant data is being passed between the components.